### PR TITLE
Refactored the structure of the custom-dashboards solution for more m…

### PR DESCRIPTION
**Pull request by etsauer**
…odularity and easier documentation and contribution

@oybed this is my first attempt that the restructure we talked about.

@tomgeorge please test and provide feedback on contribution-friendliness

@shah-zobair @raffaelespazzoli @sabre1041  FYI.

### CODE REVIEW

File: custom-dashboards/README.md [hunk 1]
```diff
@@ -1,62 +1,38 @@
...
+1. First, you must download the prerequsite repositories, [OpenShift Ansible](https://github.com/openshift/openshift-ansible), and [OpenShift Applier](https://github.com/redhat-cop/openshift-applier).
+
+        ansible-galaxy install -r requirements.yml -p galaxy
+
+
+2. (Optional) If you are running on an OpenShift Cluster older than 3.11, or your cluster did not have the monitoring stack installed at first install, you can use the following command to install the monitoring stack:
+
+        ansible-playbook -i /path/to/cluster/inventory/ monitoring.yml
+
+3. Finally, run the apply.yml playbook to deploy Grafana and all of the Dashboards.
```
Comment (oybed) [inline custom-dashboards/README.md]: Did you mean the `openshift-cluster-seed` playbook here? I don't think an `apply.yml` exists, and personally I prefer if we use the playbook shipped with the `openshift-applier`.

File: custom-dashboards/.openshift/patches/openshift-state-metrics-service-monitor.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+spec:
```
Comment (oybed) [inline custom-dashboards/.openshift/patches/openshift-state-metrics-service-monitor.yaml]: Probably should start yaml files with `---`

Comment (etsauer) [reply custom-dashboards/.openshift/patches/openshift-state-metrics-service-monitor.yaml]: @oybed added this to all yaml files, except for the ones for openshift-state-metrics as they were copied from another repo temporarily.

File: custom-dashboards/README.md (hunk 1 above)
Comment (etsauer) [reply custom-dashboards/README.md]: @oybed I removed it and updated the readme.

**Comment by tomgeorge** [comment]
LGTM

**Pull request merged**